src/main/java/com/sem/btrouble/model/SlowPowerUp.java

```diff
@@ -33,12 +33,18 @@ public SlowPowerUp() {
      * Construct power up received in the game.
      * @param xpos x position
      * @param ypos y position
+     * @param expirationTime after which the powerup expires.
      */
     public SlowPowerUp(float xpos, float ypos, int expirationTime) {
         super(xpos, ypos, expirationTime);
         on = false;
     }
     
+    @Override
+    public int hashCode() {
+      return 1;
```

Comment:
If you only return a fixed value, you break the use of any data structure that uses a hash. 
The generated hashCode function by your IDE is usually good enough, so I recommend using that.